components/org.wso2.carbon.identity.api.user.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/endpoint/impl/StartUsernamelessRegistrationApiServiceImpl.java

```diff
@@ -34,6 +34,7 @@
 import java.net.URLDecoder;
 
 import org.wso2.carbon.identity.application.authenticator.fido2.exception.FIDO2AuthenticatorClientException;
+import org.wso2.carbon.identity.application.authenticator.fido2.exception.FIDO2AuthenticatorServerException;
```

Comment:
Do we need this import now?

Reply:
Required as we catch it and throw a client exception from this class